lib/dune/getSwapMintComments.ts

```
@@ -0,0 +1,45 @@
+  };
+  const params: any = {
+    decode: "true",
+    chain_ids: `${CHAIN_ID}`,
+    topic0: ERC20_MINT_COMMENT_EVENT_SIGNATURE,
+    log_address: tokenContract,
+  };
+
+  const maximumDeep = 100;
+  let loop = 0;
+  let transactions: DuneDecodedEvent[] = [];
+  while (1) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)